sound/soc/sof/intel/hda-dsp.c

```diff
@@ -20,6 +20,17 @@
 #include "../ops.h"
 #include "hda.h"
 
+#define SOF_PM_PG_RSVD		BIT(0)
+
+/** \brief Indicates whether streaming is active */
+#define SOF_PM_PG_STREAMING	BIT(1)
+
+/** \brief Prevent power gating (0 - deep power state transitions allowed) */
+#define SOF_PM_PPG		BIT(2)
+
+/** \brief Prevent clock gating (0 - cg allowed, 1 - DSP clock always on) */
+#define SOF_PM_PCG		BIT(3)
```

Comment:
remove all those doxygen-style comments

Reply:
this should be a separate patch anyways

Reply:
> remove all those doxygen-style comments

Let me change it, it was originally from Marcin's FW PR, I put them in IPC header file.